src/js/stores/AppsStore.js

```diff
@@ -120,20 +120,55 @@ function getAppType(app) {
 function calculateTotalResources(app) {
   app.totalMem = app.mem * app.instances;
   app.totalCpus = parseFloat((app.cpus * app.instances).toPrecision(4));
-  return app;
 }
 
-function processApp(app) {
-  app = Util.extendObject(appScheme, app);
+function setHasMigrationApiSupport(app) {
```

Comment:
Sorry, I've misunderstood the naming before.
This is not a setter-function, so the naming is unfortunately misleading.
How about just `determineMigartionApiSupport()`